public/app/features/manage-dashboards/utils/validation.ts

```diff
@@ -3,12 +3,18 @@ import { getBackendSrv } from '@grafana/runtime';
 import { validationSrv } from '../services/ValidationSrv';
 
 export const validateDashboardJson = (json: string) => {
-  try {
-    JSON.parse(json);
-    return true;
-  } catch (error) {
-    return 'Not valid JSON';
+  let dashboard = JSON.parse(json);
```

Comment:
hi @iamelDuderino ✨. We need to pass the `JSON.parse(json)` through a `try {} catch {}` because if it fails the `validateDashboardJson` will throw an unexpected that's never handled and the page will fail to load.

Reply:
Hey @polibb Oops! You are correct in that the page just simply does nothing if the json is invalid. I have re-added the `try {} catch {}` and included a test for it, as well as reworded my initial errors to better align with the existing format from the catch statement. Please let me know if there is anything further, thanks!